setup.py

```diff
@@ -39,7 +39,7 @@
     install_requires=[
         'threadloop>=1,<2',
         'thrift',
-        'tornado>=4.3,<5',
+        'tornado>=4.3,<6',
```

Comment:
I am not comfortable with this change as is considering that it will force CI to install v5, meaning we'd stop testing with v4. Since Tornado is (unfortunately) a critical dependency, we need to extend CI to test with both versions, either by using tox or by manually installing 4.x before `make bootstrap`

Reply:
I agree, it's a good point.

Reply:
Done. Using `tox` is better way to solve this problem, but I think that environment matrix in .travis.yml enough for the PR.

Reply:
adding `tox` support would be great, if you feel like it (doesn't have to be this PR)